### ISSUE TITLE
Gist summaries for visually impaired surfers

### ISSUE THREAD

**Issue by Wotipati** [Wotipati/Blind_Accessibility_Papers#211]
### Links
- Paper: https://dl.acm.org/doi/10.1145/1090785.1090804

<img width="667" alt="ScreenShot 2020-05-17 15 54 04" src="https://user-images.githubusercontent.com/26996041/82137893-eda75f80-9856-11ea-8723-e0d61aadc6b7.png">

### Abstract
Anecdotal evidence suggests that Web document summaries provide the sighted reader with a basis for making decisions regarding the route to take within non-linear text; and additional research shows that sighted people use 'Gist' summaries as decision points to bolster their browsing behaviour. Other studies have found that visually impaired users are hindered in their cognition of the content of Web-pages because users must wait for an entire Web-page to be read before deciding on it's usefulness to their current task. In these cases, we draw similarities between sighted and visually impaired users, in that sighted users cannot see the target of a Web Anchor and are therefore 'handicapped'1 by the technology. Previously, we have investigate four simple summarisation algorithms against each other and a manually created summary; producing empirical evidence as a formative evaluation. This evaluation concludes that users prefer simple automatically generated 'gist' summaries thereby reducing cognitive overload and increasing awareness of the focus of the Web-page under investigation. In this paper we focus on the development of 'FireFox' based tool which creates a summary of a Web page 'on-the-fly'. The algorithm used to create this summary is based on the results of our formative evaluation which automatically and dynamically annotates Web pages with the generated 'gist' summary. In this way visually impaired users are supported in their decisions as the relevancy of the page at hand.  
- ウェブページを要約してユーザにフィードバックするシステムを提案
